plugins/Actions/ArchivingHelper.php

```diff
@@ -314,6 +315,7 @@ public static function reloadConfig()
         } else {
             self::$actionUrlCategoryDelimiter = self::$actionTitleCategoryDelimiter = $actionDelimiter;
         }
+        self::$actionTitleCategoryReverse = Config::getInstance()->General['action_title_category_reverse'];
```

Comment:
One problem that I have is that this setting would apply to all websites therefore would not be usable for many users. @mattab let's maybe discuss this issue later as it could be also interesting for our vision to manage any kind of type #7131, #4734 and especially in #7893 . I see more and more a demand for adding "settings" on a per website basis which is also needed when allowing to create any type in "Sites Management"